pkg/fanal/cache/s3.go

```diff
@@ -42,15 +41,21 @@ func (c S3Cache) PutArtifact(artifactID string, artifactConfig types.ArtifactInf
 }
 
 func (c S3Cache) DeleteBlobs(blobIDs []string) error {
-	var errs error
+	var objects []*s3.ObjectIdentifier
 	for _, blobID := range blobIDs {
 		key := fmt.Sprintf("%s/%s/%s", blobBucket, c.prefix, blobID)
-		input := &s3.DeleteBucketInput{Bucket: aws.String(key)}
-		if _, err := c.s3Client.DeleteBucket(input); err != nil {
```

Comment:
no idea why this was deletebucket when its trying to delete blobs? IMO it is incorrect when trying to delete blobs.